packages/gatsby-plugin-sass/README.md

```diff
@@ -213,6 +219,7 @@ plugins: [
 ### v3.0.0
 
 - `sass-loader` is updated to v10 which adds support for `node-sass@^5.0.0` but also switches the default `implementation` to `sass`. webpack also recommends using `sass` so this is reflected in the documentation here, too. In the [deprecation notice of node-sass](https://sass-lang.com/blog/libsass-is-deprecated#how-do-i-migrate) it is noted that switching from `node-sass` to `sass` is straightforward as both packages use the same JavaScript API.
+- all options for both `node-sass` & `sass` are moved into the `sassOptions` object
```

Comment:
Given that we know what those options are (for plugin options schema) - should we just list them here so users searching for exact option can actually find this note?

Reply:
We could link to https://github.com/sass/dart-sass/blob/master/README.md#javascript-api & https://github.com/sass/node-sass#options or what did you have in mind?

Reply:
That's good approach too 👍